client/src/components/App.js

```diff
@@ -31,10 +32,11 @@ const App = (props) => {
         <Route exact path="/users/new" component={RegistrationForm} />
         <Route exact path="/user-sessions/new" component={SignInForm} />
         <Route exact path="/podcasts" component={PodcastsIndex} />
+        <Route exact path="/podcasts/:id" component={PodcastShowPage} />
         <Route exact path="/genres" component={GenreIndex} />
       </Switch>
     </Router>
-  )
-}
+  );
+};
 
-export default hot(App)
+export default hot(App);
```

Comment:
Did you all as a group decide to go back to semi colons?  Not a wrong thing, but I just want to make sure the whole team is consistent with styling